docs/open-source.md

```diff
@@ -15,6 +15,7 @@ CloudEvents v1.0, please include it in the list below.
 - [CloudEvents Verify](https://github.com/btbd/CEVerify): is a tool to help
   verify CloudEvents according to the proper specifications. It is currently
   being hosted publicly [here](http://soaphub.org/ceverify).
+- [Cloud Shapes](https://github.com/neuroglia-io/cloud-shapes): Cloud Shapes is an event-driven database designed for real-time projection materialization based on CloudEvents.
```

Comment:
minor thing, but can you wrap this at 80 columns to match the rest of the doc?

Actually, while you're in here, can you wrap lines 19, 25 and 26 too? :-)

Reply:
Went ahead and did that in #1329